lessons/git/intro/slides-2017.Rmd

```diff
@@ -0,0 +1,305 @@
+---
+title: "Intro to Git"
+author: "Shyam Srinivasan"
+date: "October 5, 2017"
+output: beamer_presentation
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+# Learning expectations
+
+This won't even touch the surface of git... But! Hopefully, you:
+
+- Recognize the power of using version control
+- Know the basic tools to get started using git
+- Where to go for help
+
+But! No expectation to actually start using it :)
+
+
+# Getting started with Git and GitHub
+
+Installing and running Git and GitHub
+
+## Linux
+https://www.howtoforge.com/tutorial/install-git-and-github-on-ubuntu-14.04/
+
+* Dowload and Install Git for Linux
+`sudo apt-get install git`
+
+## Windows/MacOS
+https://desktop.github.com/
+
+* Make sure you install posh-git along with GitHub
+
+# Configuring your setup:
+
+```bash
+git config --global user.name "Your Name"
+git config --global user.email "you@some.domain"
+```
+
+Optional Configurations
+```bash
+git config --global color.ui "auto" 
+```
+
+Setup Editor for Text Files
+
+```bash
+git config --global core.editor "your_editor"
+git config --list # To confirm
+```
+
+# Creating a Repo on Github
+The cloud (GitHub) serves as an online repo where files are accessible 24x7
+
+* Go to [GitHub](https://github.com/)
+
+* Create a new repo by pressing the `New Repository` button
+
+* Let us keep the name short and self explanatory and call it 'git-training' 
```

Comment:
The shell script forks `studyGroup` and/or uses `sandbox`; needs to be consistent.

Reply:
This has been corrected. sandbox repo used everywhere. All references to studyGroup have been removed

Reply:
This still uses 'git-training'.